src/MultiFormGoals/resources/js/blocks/progress-bar/edit/inspector.js

```diff
@@ -40,12 +41,11 @@ const Inspector = ( { attributes, setAttributes } ) => {
 					onChange={ ( value ) => saveSetting( 'goal', value ) }
 					value={ goal }
 				/>
-				<TextControl
+				<DateTimeControl
 					name="enddate"
 					label={ __( 'Goal end date', 'give' ) }
-					type="date"
-					onChange={ ( value ) => saveSetting( 'enddate', value ) }
 					value={ enddate }
+					onChange={ ( value ) => saveSetting( 'enddate', value ) }
```

Comment:
@henryholtgeerts we should update the `enddate` attribute to reflect the date/time update, perhaps `deadline`.

Reply:
Personally, I feel that "End date" is still an appropriate name for this control/attribute. Given that many users will never actually change the time itself, I think it still communicates its purpose while the control allows for a greater level of resolution (if users want it). 

Additionally, the term "End date" is used in GiveIO, where users are still granted the ability to specify a time for the campaign to end.

Reply:
Sure. That works. Not going to hold up the PR.